docs/maintenance.md

```diff
@@ -0,0 +1,81 @@
+Maintenance guide
+=================
+
+This is the maintenance guide for pvc-autoresizer.
+
+How to upgrade supported Kubernetes version
+-------------------------------------------
+
+pvc-autoresizer depends on some Kubernetes repositories like `k8s.io/client-go` and should support 3 consecutive Kubernets versions at a time.
+Here is the guide for how to upgrade the supported versions.
+Issues and PRs related to the last upgrade task also help you understand how to upgrade the supported versions,
+so checking them together with this guide is recommended when you do this task.
```

Comment:
How about adding the link to this PR?

https://github.com/topolvm/pvc-autoresizer/pull/85